core_contracts/governance/governance.py

```diff
@@ -778,5 +778,9 @@ def fallback(self):
         pass
 
     @eventlog(indexed=2)
-    def VoteCast(self, vote_name: str, vote: bool, voter: Address, stake: int, total_for: int, total_against: int):
+    def VoteCast(self, vote_index: int, vote: bool, voter: Address, stake: int, total_for: int, total_against: int):
```

Comment:
I think it may be better to leave this as is. Either that or change it to a different eventlog name, like VoteCast_v2. Please discuss with Ajaya. for dividends he made a new eventlog when the parameters changed.

Reply:
Let's just skip the change to the VoteCast method for now.